plugins/pdns/pdns_rel

```diff
@@ -31,17 +36,19 @@ if [ "$1" = "config" ]; then
         exit 0
 fi
 
-hits=$($command packetcache-hit | awk -F= '{print $2}')
-queries=$($command udp-queries | awk -F= '{print $2}')
-old_hits=$(cat $state_file | head -n1)
-old_queries=$(cat $state_file | tail -n1)
+hits=$($command packetcache-hit)
+queries=$($command udp-queries)
+if [ -f $state_file ]; then
```

Comment:
Please quote variables in order to avoid whitespace issues.
Maybe you want to try `shellcheck` - it is a great tool for verifying the intricate details and pitfalls of shell syntax.